getting-started/templates/Openshift-GCP/os_gcp_supplemental_values.yaml

```diff
@@ -0,0 +1,286 @@
+global:
+  ingress:
+    api:
+      annotations:
+        kubernetes.io/ingress.class: <user-defined-ingress-class> #<ATTENTION> Enter the volume name where the token is mounted
+    ui:
+      annotations:
+        kubernetes.io/ingress.class: <user-defined-ingress-class> #<ATTENTION> Enter the volume name where the token is mounted
+
+rabbitmq:
+  global:
+    compatibility:
+      openshift:
+        adaptSecurityContext: disabled
+
+webserver:
+  redis-cluster:
+    global:
+      compatibility:
+        openshift:
+          adaptSecurityContext: disabled
+
+testmonitorservice:
+  ## Configuring sidecars for the Test Monitor Service
+  sidecars:
+    - name: cloud-sql-auth-proxy
+      image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.8.0
+      volumeMounts:
+        - name: <token-volume-name> #<ATTENTION> Enter the volume name where the token is mounted
+          readOnly: true
+          mountPath: /var/run/secrets/openshift/serviceaccount/
+        - name: <config-volume-name> #<ATTENTION> Enter the volume name where config.json is mounted
+          mountPath: /secrets/
+          readOnly: true
+      env:
+        - name: "GOOGLE_APPLICATION_CREDENTIALS"
+          value: /secrets/service_account.json
+      args:
+        # If connecting from a VPC-native GKE cluster, you can use the
+        # following flag to have the proxy connect over private IP
+        - "--private-ip"
+        # - "--auto-iam-authn"
+
+        # Enable structured logging with LogEntry format:
+        - "--structured-logs"
+        - "--exit-zero-on-sigterm"
+
+        # Replace DB_PORT with the port the proxy should listen on
+        - "--port=5432"
+        - "<connection-name>" #<ATTENTION> Enter the connection name from the CloudSQL instance
+      # - "--credentials-file=/var/run/secrets/openshift/serviceaccount/token"
+      securityContext:
+        # The default Cloud SQL Auth Proxy image runs as the
+        # "nonroot" user and group (uid: 65532) by default.
+        runAsNonRoot: true
+      # restartPolicy: Always
```

Comment:
Is there a reason this is commented out? It needs to be set for the sidecar to function properly.

Reply:
The 'restartPolicy' has been uncommented